packages/sdk-js/src/auto-wrapper.ts

```diff
@@ -205,6 +209,46 @@ function getAttributes() {
   return attributes;
 }
 
+function setAntiFlicker() {
+  window.clearTimeout(antiFlickerTimeout);
+
+  const timeoutMs =
+    windowContext.antiFlickerTimeout ??
+    (dataContext.antiFlickerTimeout
+      ? parseInt(dataContext.antiFlickerTimeout)
+      : null) ??
+    3500;
```

Comment:
If `dataContext.antiFlickerTimeout` is not parseable as an int (e.g. `"a"`), this will result in `NaN`.  This means, the delay will be treated as `0ms` instead of `3500ms`.

Reply:
that seems like a user error, but can add some guards